src/main/java/seedu/address/logic/parser/AssignCommandParser.java

```diff
@@ -30,31 +32,31 @@
     public AssignCommand parse(String args) throws ParseException {
```

Comment:
We can afford a SuppressWarnings here since we are checking with our custom function.

Reply:
```suggestion
    @SuppressWarnings("OptionalGetWithoutIsPresent") // This is checked using our custom function (arePrefixesPresent).
    public AssignCommand parse(String args) throws ParseException {
```